src/EntryPoint/RestEntryPoints.ts

```diff
@@ -51,11 +51,39 @@ export class RestEntryPoints implements EntryPoints {
     });
   }
 
-  public async entrypoints(projectId: string): Promise<EntryPoint[]> {
-    const res = await this.client.get(
-      `/api/v2/projects/${projectId}/entry-points`
-    );
+  public async entrypoints(
+    projectId: string,
+    limit: number = 10
+  ): Promise<EntryPoint[]> {
+    let l = limit;
```

Comment:
Use more descriptive name for such variables
```suggestion
    let remaining = limit;
```